plugins/duo_admin/.CHECKSUM

```diff
@@ -49,7 +49,7 @@
 		},
 		{
 			"identifier": "monitor_logs/schema.py",
-			"hash": "5ce8b19344da9506c08512370681f783"
+			"hash": "4119a8c82613406e16d830d7b48e0c86"
```

Comment:
there doesn't look to be any changes to `monitor_logs/schema.py`, was updating this hash intentional ?